data_steward/analytics/rt_cdr_qc/cdr_deid_qa_report2_row_suppression.py

```diff
@@ -20,18 +20,29 @@
 
 import urllib
 import pandas as pd
+from utils import auth
+from gcloud.bq import BigQueryClient
+from analytics.cdr_ops.notebook_utils import execute, IMPERSONATION_SCOPES
 pd.options.display.max_rows = 120
 
 # + tags=["parameters"]
 # Parameters
 project_id = ""
 deid_cdr = ""
-com_cdr ="" 
+com_cdr =""
```

Comment:
I had to select text to see the whitespace. Since it's being changed anyway, I'd add a nit to run yapf on this to add a space after `=` and any other formatting changes.